src/main/java/org/jitsi/videobridge/Videobridge.java

```diff
@@ -99,6 +99,20 @@
         = "org.jitsi.videobridge.MEDIA_RECORDING_TOKEN";
 
     /**
+     * If an endpoint is provided at time of channel creation, use the
+     * endpoint for the channel id
+     */
+    public static final String USE_ENDPOINT_FOR_CHANNEL_ID
+            = "org.jitsi.videobridge.useEndpointForChannelId";
```

Comment:
Can we rename that to org.jitsi.videobridge.USE_ENDPOINT_FOR_CHANNEL_ID for consistency? I know some of the REST properties use camel case, but everything else uses all caps with underscores.